cpp/src/barretenberg/stdlib/merkle_tree/membership.test.cpp

```diff
@@ -20,9 +20,7 @@ using namespace barretenberg;
 using namespace proof_system::plonk::stdlib::merkle_tree;
 using namespace plonk::stdlib;
 
-using Composer = plonk::UltraComposer;
-using Prover = plonk::UltraProver;
-using Verifier = plonk::UltraVerifier;
+using Composer = proof_system::UltraCircuitConstructor;
```

Comment:
I suppose this aliasing is just maintained for convenience and once we do away with the Composer it can be updated to `Constructor` or whatever?

Reply:
Yes